doc/modules/model_evaluation.rst

```diff
@@ -1027,6 +1034,12 @@ Similarly, labels not present in the data sample may be accounted for in macro-a
   >>> metrics.precision_score(y_true, y_pred, labels=[0, 1, 2, 3], average='macro')
   0.166...
 
+.. topic:: References:
+
+    .. [OB2019] Opitz, J., & Burst, S. (2019). `Macro f1 and macro f1
```

Comment:
Nowadays, we have an arxiv integration, e.g.:

```
(1) :arxiv:`"Finding structure with randomness:
    Stochastic algorithms for constructing approximate matrix decompositions."
    <0909.4061>`
```

Reply:
Done!